website/content/api-docs/secret/aws.mdx

```diff
@@ -670,6 +670,10 @@ This endpoint creates or updates static role definitions. A static role is a 1-t
 with an AWS IAM User, which will be adopted and managed by Vault, including rotating it according
 to the configured `rotation_period`.
 
+In addition, this endpoint supports cross account management.
+Vault will use the credentials obtained by assuming roles in another AWS account to perform AWS operations.
```

Comment:
```suggestion
In addition, this endpoint supports cross-account management.
Vault will use the credentials obtained by assuming a role in another AWS account to perform AWS operations.
```
I think it should be singular because a static role is only configured to assume 1 role each.